content/docs/create-a-new-react-app.md

```diff
@@ -8,87 +8,87 @@ prev: add-react-to-a-website.html
+Gunakan *toolchains* yang terintegrasi untuk pengalaman pengguna dan pengembang yang terbaik.
 
-This page describes a few popular React toolchains which help with tasks like:
+Halaman ini menjelaskan beberapa *toolchains* yang bisa membantu menyelesaikan tugas seperti:
 
-* Scaling to many files and components.
-* Using third-party libraries from npm.
-* Detecting common mistakes early.
-* Live-editing CSS and JS in development.
-* Optimizing the output for production.
+* Penskalaan ke banyak *file* dan komponen.
+* Penggunaan *library* pihak ketiga dari *npm*.
+* Mendeteksi kesalahan umum lebih awal.
+* Pengeditan *CSS* dan *JS* secara langsung dalam tahap pengembangan.
+* Mengoptimalkan keluaran untuk tahap produksi.
 
-The toolchains recommended on this page **don't require configuration to get started**.
+*Toolchains* yang direkomendasikan pada halaman ini **tidak membutuhkan konfigurasi untuk memulai**.
 
-## You Might Not Need a Toolchain {#you-might-not-need-a-toolchain}
+## Anda mungkin tidak butuh *Toolchains* {#you-might-not-need-a-toolchain}
 
-If you don't experience the problems described above or don't feel comfortable using JavaScript tools yet, consider [adding React as a plain `<script>` tag on an HTML page](/docs/add-react-to-a-website.html), optionally [with JSX](/docs/add-react-to-a-website.html#optional-try-react-with-jsx).
+Jika anda tidak mengalami masalah yang dijelaskan di atas atau belum merasa nyaman menggunakan *tools* dari *JavaScript*, pertimbangkan untuk [menambahkan *React* sebagai *tag* `<script>` biasa pada halaman *HTML*](/docs/add-react-to-a-website.html). Anda dapat melakukan itu [dengan *JSX*](/docs/add-react-to-a-website.html#optional-try-react-with-jsx).
 
-This is also **the easiest way to integrate React into an existing website.** You can always add a larger toolchain if you find it helpful!
+Cara tersebut juga merupakan **cara termudah untuk mengintegrasikan *React* ke dalam *website* yang sudah ada.** Anda bisa menambah *Toolchains* yang lebih besar kapanpun anda inginkan jika anda merasa itu dapat membantu!
 
-## Recommended Toolchains {#recommended-toolchains}
+## *Toolchains* Yang Direkomendasikan {#recommended-toolchains}
 
-The React team primarily recommends these solutions:
+Tim *React* sangat merekomendasikan solusi ini:
 
-- If you're **learning React** or **creating a new [single-page](/docs/glossary.html#single-page-application) app,** use [Create React App](#create-react-app).
-- If you're building a **server-rendered website with Node.js,** try [Next.js](#nextjs).
-- If you're building a **static content-oriented website,** try [Gatsby](#gatsby).
-- If you're building a **component library** or **integrating with an existing codebase**, try [More Flexible Toolchains](#more-flexible-toolchains).
+- Jika anda **Sedang belajar *React*** atau **membuat Aplikasi [single-page](/docs/glossary.html#single-page-application) baru,** gunakan [*Create React App*](#create-react-app).
```

Comment:
Sedang & Aplikasi shouldn't be capitalized 😄

Reply:
```suggestion
- Jika anda **sedang belajar *React*** atau **membuat aplikasi [single-page](/docs/glossary.html#single-page-application) baru,** gunakan [*Create React App*](#create-react-app).
```